madvaultllc/slswc-client
# Address changes requested in code review

Closes #17

**Proposed changes**
Address the issues highlighted in #17

- [ ] Convert license activate/deactivate to button. Closes #16
- [ ] Convert licenses tab to cards. Closes #20 
- [ ] Make environment option so it can be detected by server. Closes #18 
- [ ] Convert to composer package

**How to test**
Install using `composer install madvault/slswc-client`
- Check the `SLSWC_Client` and `SLSWC_Client_Manager` classes are available in the `Madvault\Slswc\Client` namespace.
- Check Licenses tab in Dashboard > Settings > License Manager page and check:
1. New card layout
2. Activate/Deactivate button
3. Test that you can activate/deactivate a license using the new client

Check against each of the issues linked above
